docs/developer-docs/multi-chain/overview.mdx

```diff
@@ -0,0 +1,113 @@
+---
+keywords: [advanced, bitcoin, btc, ckbtc, overview, multichain, multi-chain, ethereum, evm, eth, cketh]
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Overview
+
+<MarkdownChipRow labels={["Advanced", "Multi-chain", "Concept" ]} />
+
+## What is multi-chain?
+
+The term 'multi-chain' refers to the Internet Computer's ability to trustlessly make direct calls to another blockchain, enabling dapps to expand their capabilities across multiple networks. ICP smart contracts can interact with another chain's smart contracts and native assets. Multi-chain introduces the decentralized custody of assets. Smart contracts can have multiple addresses for receving and holding assets of other chains. 
+
+By interacting with assets on other chains, smart contracts can unleash unprecedented possibilities by tapping into the unbounded scalability of IC with extremely low costs, while still preserving the highest degree of security. This ability to communicate directly with other blockchain networks is possible using the protocol's unique cryptography features, which support decentralized signing of transactions that natively execute on other networks. ICP currently has multi-chain integrations with the Bitcoin and Ethereum networks.
```

Comment:
Sam and Ulan suggested to stay away from the marketing language. Sorry for the confusion.

```suggestion
By interacting with assets on other chains, smart contracts can unleash new possibilities by tapping into the fast finality, low costs and reverse gas model of the IC, while still preserving the highest degree of security. This ability to communicate directly with other blockchain networks is possible using the protocol's unique cryptography features, which support decentralized signing of transactions that natively execute on other networks. ICP currently has multi-chain integrations with the Bitcoin and Ethereum networks.
```